Cue/app/tabs/library/LibraryHome.ios.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import React from 'react'
+import { View, Text, TouchableOpacity, Navigator, Platform, AlertIOS } from 'react-native'
+
+import { connect } from 'react-redux'
+
+import type { Deck } from '../../api/types'
+
+import CueHeader from '../../common/CueHeader'
+import CueIcons from '../../common/CueIcons'
+
+import LibraryListView from './LibraryListView'
+import { createDeck, loadLibrary, syncLibrary } from '../../actions/library'
+
+const styles = {
+  container: {
+    flex: 1,
+    backgroundColor: 'white'
+  },
+  bodyContainer: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center'
+  }
+}
+
+type Props = {
+  decks: Array<Deck>,
+  navigator: Navigator,
+  onPressMenu?: () => void
+}
+
+class LibraryHome extends React.Component {
+  props: Props
+
+  state: {
+    refreshing: boolean
+  }
+
+  _refresh = () => {
+    this.setState({
+      ...this.state,
+      refreshing: true
+   });
+   this.props.onLoadLibrary(this.props.localChanges);
+  }
+
+  componentWillReceiveProps(newProps: Props) {
+    this.setState({
+      ...this.state,
+      refreshing: false
+    });
+  }
+
+  constructor(props: Props) {
+   super(props);
+   this.state = {
+     refreshing: false
+   };
+   this._refresh = this._refresh.bind(this)
```

Comment:
Might be because `_refresh` is only declared as an arrow function in `LibraryHome.ios`, not in `LibraryHome.android`.